### PR TITLE
Remove Units from Report Strings

### PR DESCRIPTION
**Pull request by MarcusHolly**
## Summary/Motivation:
Gets rid of units that are explicitly stated in the strings of reports

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/unit_models/nanofiltration_DSPMDE_0D.py [hunk 1]
```diff
@@ -1579,7 +1579,7 @@ def _get_performance_contents(self, time_point=0):
             time_point
         ].flow_vol_phase["Liq"]
 
-        expr_dict["Average Volumetric Flux (LMH)"] = (
+        expr_dict["Average Volumetric Flux"] = (
             self.flux_vol_water_avg[time_point] * 3.6e6
```
Comment (kurbansitterley) [inline watertap/unit_models/nanofiltration_DSPMDE_0D.py]: I believe this needs the same treatment from #1424 (remove the hardcoded *3.6e6)

**Review by kurbansitterley — CHANGES_REQUESTED** [review]
just the change to NF and should be good

File: watertap/unit_models/nanofiltration_DSPMDE_0D.py (hunk 1 above)
Comment (MarcusHolly) [reply watertap/unit_models/nanofiltration_DSPMDE_0D.py]: Thanks for catching this!

**Review by kurbansitterley — APPROVED** [review]
easy enough, LGTM

File: watertap/unit_models/MD/membrane_distillation_base.py [hunk 2]
```diff
@@ -696,7 +696,7 @@ def _get_performance_contents(self, time_point=0):
         if hasattr(self, "width"):
             var_dict["Membrane Width"] = self.width
 
-        expr_dict["Average Solute Flux (LMH)"] = self.flux_mass_avg[time_point] * 1000
+        expr_dict["Average Solute Flux"] = self.flux_mass_avg[time_point] * 1000
```
Comment (hunterbarber) [inline watertap/unit_models/MD/membrane_distillation_base.py]: @zacharybinger Was the conclusion that the multiplication of the pyomo variables with units by a float actually doesn't do anything in the printing, or these still correctly print the value in the appropriate units?

**Review by TimBartholomew — APPROVED** [review]
LGTM

**Pull request merged**